news/_posts/releases/2020-03-04-prestashop-1-7-6-4-maintenance-release.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title:  "Release Of PrestaShop 1.7.6.4"
+subtitle: "MAINTENANCE VERSION OF THE 1.7.6.X BRANCH"
+date:   2020-03-04 10:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2020/03/meta-1764.png
+icon: icon-checked
+tags:
+ - 1.7
+ - 1.7.6.x
+ - version
+ - patch
+ - releases
+---
+
+After a few months dedicated to fixing a security issue and all regressions found on the previous 1.7.6 versions, PrestaShop 1.7.6.4 is finally available! 
+
+![1.7.6.4 is available!](/assets/images/2020/03/1.7.6.4_banner.jpg)
+
+This maintenance release fixes 9 issues reported on the versions between 1.7.6.0 to the 1.7.6.3.
+Since this version fixes a security problem, we **strongly** encourage you to upgrade your shop as soon as possible.
+We suggest upgrading your shop quickly to benefit from these fixes. Of course, don’t forget to backup before.
+
+The security patch protects your shop against attacks aimed at modifying your customers' personal information.
+
+Reminder:  the 1-Click Upgrade module’s latest version is v4.10.1, don’t forget to upgrade it.
+
+## Main fixes
+Below are listed some of the main regression issues we fixed in this version :
+
+* An error case on the customer email validation ([#17809](https://github.com/PrestaShop/PrestaShop/issues/17809))
+* Failed load the product_page.css and translations.css files ([#17584](https://github.com/PrestaShop/PrestaShop/issues/17584))
+* Display issues on the SEO live previews on Safari desktop ([#17525](https://github.com/PrestaShop/PrestaShop/issues/17525))
+* A combination color wasn’t displayed when it was selected ([#17437](https://github.com/PrestaShop/PrestaShop/issues/17437))
+* Couldn’t view a category if all subcategories were disabled ([#17379](https://github.com/PrestaShop/PrestaShop/issues/17379))
+* Some products combination wasn’t displayed depending on the stock and the product configuration ([#17345](https://github.com/PrestaShop/PrestaShop/issues/17345)) 
```

Comment:
Some product combinations were not displayed on particular stock and product configurations